lib/bugsnag/http_client/httpoison.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Bugsnag.HTTPClient.HTTPoison do
```

Comment:
What are your thoughts on namespacing these under `Bugsnag.HTTPClient.Adapters` or maybe suffixing them, like `HTTPoisonAdapter`, `TeslaAdapter` and so on?